### ISSUE TITLE
RadListView crashes on rotate orientation, on iOS, Vue (when using item layout Grid or Staggered)

### ISSUE THREAD

**Issue by tralves** [NativeScript/nativescript-ui-feedback#1056]
### Tell us about the problem

Using a RadListView with `layout="grid"`, on a NS-Vue project, on iOS, the app crashes when changing orientation.

### Which platform(s) does your issue occur on?

iOS. I only tested on the simulator, though.

### Please provide the following version numbers that your issue occurs with:
- **Progress NativeScript UI plugin version:** 6.0.0
- **CLI:** 5.1.1
- **Cross-platform modules:**5.2.2
- **Runtime(s):**  5.0.0

### Please tell us how to recreate the issue in as much detail as possible.

I created a project with the simplest example possible [here](https://github.com/tralves/ns-ui-listvue-ios-crash-example).

1. Start the app on iOS (`tns run ios --bundle`).
2. Rotate the device or simulator.

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.

I tested with the code from the documentation:

```
      <RadListView for="item in itemList"
                   layout="grid"
                   itemHeight="100">
        <v-template>
          <StackLayout orientation="vertical">
            <Label :text="item.name"></Label>
          </StackLayout>
        </v-template>
      </RadListView>
```

**Comment by NickIliev** [comment]
Confirming this one as a bug - the issue is appearing when using item layout of type `grid` or of type `staggered`.

Removing the item layout property is resolving the case.

**Comment by eggin** [comment]
It's not just on Vue projects. I'm experiencing this on a vanilla NS Typescript project too.

**Comment by dottodot** [comment]
Same is happening for Angular as well. Any ideas on when this will be fixed?

**Comment by dottodot** [comment]
Actually seems to be ok in the latest release which is a bit odd considering the issue hasn't be addressed.

**Comment by tralves** [comment]
I just tested both on my project and the [test project](https://github.com/tralves/ns-ui-listvue-ios-crash-example) and it still crashes.

I updated the ns-ui-listview to 6.2.0.

**Comment by dottodot** [comment]
@tralves You are right, however I do have one view that isn't crashing, the only difference I can see is it doesn't have any images in it.

**Comment by tralves** [comment]
My sample app doesn't have images either... can you show me your sample that doesn't crash?

**Comment by dottodot** [comment]
@tralves 

```
<GridLayout class="page">
  <RadListView class="m-5" [items]="projects$ | async" backgroundColor="transparent"
    (itemLoading)="makeBackgroundTransparent($event)">
    <ng-template tkListItemTemplate let-project="item">
      <StackLayout appCard class="m-5" (tap)="select(project.id)">
        <GridLayout class="list-group-item" rows="*,*,*" columns="auto, *" card-body>
          <Label class="h3" row="0" col="1" [text]="project.title"></Label>
          <Label class="text-uppercase h6" row="1" col="1" [text]="project.site_code"></Label>
          <Label class="text-uppercase h6" row="2" col="1" [text]="project.project_type"></Label>
        </GridLayout>
      </StackLayout>
    </ng-template>
    <ListViewStaggeredLayout tkListViewLayout scrollDirection="Vertical" spanCount="4">
    </ListViewStaggeredLayout>
  </RadListView>
</GridLayout>
```

**Comment by matecode** [comment]
Anyone handling this bug? It's preventing our project from being updated to current plugin version.

**Comment by YaakovDavid** [comment]
Same issue, with ``` ListViewGridLayout``` and ``` ListViewStaggeredLayout``` that the app crashes on rotation this is both on the simulator and running on any iOS device (through TestFlight). I do have images in the GridLayout but even when I comment it out it still crashes. I am using NativeScript core with Typescript. Any fix/ workaround will be extremely helpful.

**Comment by silviosan** [comment]
Are there any news regarding this severe bug? Any information if and when this will be fixed is really appreciated.

**Comment by VladimirAmiorkov** [comment]
Hi,

We are currently investigating the source of this issue and hope to have a patch fix as soon as possible.

**Comment by fhackenb** [comment]
I've been seeing an error in sentry sporadically for a couple months now: 
```
EXC_BAD_ACCESS: Exception 1, Code 1863350352, Subcode 8 >
Stack overflow in (null)
  ?, in CFBasicHashCreateCopy
  ?, in CFDictionaryCreateMutableCopy
  ?, in -[_CFXNotificationNameWildcardObjectRegistration find:matching:]
  ?, in -[_CFXNotificationRegistrar find:object:observer:enumerator:]
  ?, in _CFXNotificationPost
...
(95 additional frame(s) were not displayed)
```
Just recently realized it was tied to this orientation change issue. Glad to hear a fix is on the way

**Comment by zbranzov** [comment]
The fix is released in nativescript-ui-listview@6.4.2

**Comment by YaakovDavid** [comment]
It seems to work, Thank you @VladimirAmiorkov  @zbranzov

**Comment by YaakovDavid** [comment]
It works great with  ```nativescript-ui-listview@6.4.2``` but I just upgraded to NativeScript 6.0 with ```npm install -g nativescript@rc``` and it changed the list view to  ```"nativescript-ui-listview": "7.0.0-androidx-110619",``` in my package.json which broke the listview on rotation again. Can you move the fix the Nativescript 6.0 as well please.

**Comment by VladimirAmiorkov** [comment]
Hi @YaakovDavid ,

We are in the process of updating the `7.0.0-androidx` tag with our latest official fixes so stay tuned.
